src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,33 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'grasp')
+def fuze_grasp(robot, fuze, manip=None):
```

Comment:
Is this independent of the rest of the pull request? When would we want to use `TSRLibrary` to perform a grasp on a Fuze bottle versus the push-grasp?